Processes/How-To-Submit-Content-to-the-Docs-Portal.md

```diff
@@ -0,0 +1,26 @@
+## How To Submit Content to EOS Documentation Portal
+
+**Scenario**: You have written content to add to the [EOS Documentation Portal](https://docs.eosnetwork.com) in .md format. Now what do you need to do?
+
+Complete the following steps to submit your content to DevRel:
```

Comment:
Prepend this stem sentence with a section titled "Steps"

Reply:
Done